src/utils/gitExtension.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+import * as semver from 'semver';
+import {API, GitExtension, Ref, Repository} from './git';
+
+function getGitExtensionAPI(): API {
+   return (<GitExtension>(
+      vscode.extensions.getExtension('vscode.git')!.exports
+   )).getAPI(1);
+}
+
+export function getRepositories(): Repository[] {
+   const gitAPI = getGitExtensionAPI();
+   return gitAPI.repositories;
+}
+
+export async function getBranches(repository: vscode.Uri): Promise<Ref[]> {
```

Comment:
💡 This function seems to be never used and only ported because the interface has the usage, what is the future use of this? Thanks.

Reply:
used within the draft workflow experience to populate the branch selection prompt